devicetypes/smartthings/zigbee-lock.src/zigbee-lock.groovy

```diff
@@ -189,7 +201,19 @@ def refresh() {
  */
 def configure() {
 	log.trace "ZigBee DTH - Executing configure() for device ${device.displayName}"
-	def cmds = doConfigure()
+
+	// Device-Watch allows 2 check-in misses from device + ping (plus 10 min lag time)
+	sendEvent(name: "checkInterval", value: 2 * 60 * 60 + 10 * 60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID, offlinePingable: "1"])
+
+	def cmds = []
+	if (isC2OLock()) {
+		cmds += zigbee.readAttribute(CLUSTER_DOORLOCK, DOORLOCK_ATTR_LOCKSTATE)
+		cmds += zigbee.readAttribute(CLUSTER_IAS_ZONE, IAS_ATTR_ZONE_STATUS)
+		cmds += zigbee.enrollResponse()
+		cmds += zigbee.iasZoneConfig(30, 60 * 5)
```

Comment:
We should actually stop using the `iasZoneConfig` command as there is an unnecessary config as a part of it that results in a lot of extra executions.  You can replace this with:

```
cmds += configureReporting(CLUSTER_IAS_ZONE, IAS_ATTR_ZONE_STATUS, DataType.BITMAP16, 30, 60*5, null)
```